docker-compose.link.yaml

```diff
@@ -0,0 +1,4 @@
+networks:
+  default:
+    external:
+      name: ffc-sfd
```

Comment:
Is this the correct network name? Saw `fcp-sfd` used on other repo

Reply:
Should be ffc-sfd so can all run in local Docker network together